cosmos/sql/tools/chaos/cosmosdb_client_network_chaos_tool/readme.md

```diff
@@ -2,7 +2,10 @@
 
 The Cosmos DB client-side network chaos tool is a set of scripts that allows you to simulate network chaos scenarios for Azure Cosmos DB clients. It is designed to help you test the resilience and performance of your Cosmos DB applications under various network conditions. These conditions will only be experienced by the clients that you wish to target for the resiliency test. The rest of the clients and the Cosmos DB service itself won't notice any disruptions from the use of this tool.
 - <span style="color:red">**Warning**</span>: 
-    - This tool disrupts the network traffic between the specified target VM(s)/VMSS and the specified Cosmos DB collection/container for the specified account within the specified region but it can potentially affect multiple Cosmos DB accounts in the region. It may also affect other applications running on the specified target VM(s)/VMSS.
+    - This tool disrupts the network traffic between the specified target VM(s)/VMSS and the specified Cosmos DB collection/container for the specified account within the specified region.
+        - If public access is enabled for the Cosmos DB account, the tool can potentially affect multiple Cosmos DB accounts in the region. 
+        - If only private endpoint is enabled while the public access is disabled, then, the network disruption will only affect other collections in the account. 
+    - The tool may also disrupt network traffic for other applications running on the specified target VM(s)/VMSS.
```

Comment:
This is not clear. Not sure if we need this.

Reply:
Removed.